src/test/test_timer.cpp

```diff
@@ -24,7 +24,7 @@ SISL_OPTION_GROUP(test_timer,
                    ::cxxopts::value< uint32_t >()->default_value("4"), "number"),
                   (user_threads, "", "user_threads", "user_threads", ::cxxopts::value< uint32_t >()->default_value("2"),
                    "number"),
-                  (num_timers, "", "num_timers", "num_timers", ::cxxopts::value< uint64_t >()->default_value("1000"),
+                  (num_timers, "", "num_timers", "num_timers", ::cxxopts::value< uint64_t >()->default_value("100"),
```

Comment:
Isn't better to cancel timers created in `global_recurring_timer` in tearDown instead of reducing the `num_timers` here?

Reply:
i am not quite familiar with the timer here. please make the change, and i will close this pr

Reply:
@shosseinimotlagh 

before `timer_parallel_to_shutdown` is completed and tearDown is called, above will happen. so we`d better to reduce `num_timers` here